client/src/containers/backend/Dashboard.js

```diff
@@ -19,7 +19,7 @@ const perPage = 5;
 export class DashboardContainer extends PureComponent {
   static fetchData = (getState, dispatch) => {
     const projectsRequest = request(
-      projectsAPI.index({}, { size: perPage }),
+      projectsAPI.index({}, { size: perPage }, true),
```

Comment:
Why this change? As far as I can tell, the method signature for the index method is: `index(filter = {}, page = {})`, yet you're passing a third argument to it.